app/presenters/account_show_presenter.rb

```diff
@@ -63,7 +63,16 @@ def pending_idv?
   end
 
   def pending_ipp?
-    !!user.pending_profile&.in_person_verification_pending?
+    in_person_enrollment = user&.in_person_enrollments&.first
+    if in_person_enrollment&.expired?
+      user.pending_in_person_enrollment.present?
+    elsif in_person_enrollment&.cancelled?
+      user.pending_in_person_enrollment.present?
+    elsif in_person_enrollment&.failed?
+      user.pending_in_person_enrollment.present?
+    else
+      !!user.pending_profile&.in_person_verification_pending?
+    end
```

Comment:
Maybe simpler option to check that the enrollment associated with the pending profile is explicitly pending?

```suggestion
    !!user.pending_profile&.in_person_enrollment&.pending?
```

Reply:
Thanks for the feedback. 🙏🏻 I am going to close this PR because, as Gina mentioned [here](https://github.com/18F/identity-idp/pull/11129#issuecomment-2315674140), Joy decided to backfill the data.  This PR is no longer relevant.

(I am going to merge the regression specs in a separate PR.)